src/antares/model/area.py

```diff
@@ -372,3 +372,6 @@ def create_hydro(
         hydro = self._area_service.create_hydro(self.id, properties, matrices)
         self._hydro = hydro
         return hydro
+
+    def read_load(self) -> Load:
```

Comment:
There's already a method named `get_load_matrix` inside area you should use this and just put your code inside the local implementation